.gitignore

```diff
@@ -39,3 +39,4 @@ bower.json
 
 # Ignore Byebug command history file.
 .byebug_history
+/config/database.yml
```

Comment:
fwiw, this file should really be checked in with the secret stuff set via `Rails.application.secrets`